vaadin-select-flow-integration-tests/pom.xml

```diff
@@ -85,6 +85,17 @@
             <version>${flow.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>vaadin-button-flow</artifactId>
+            <version>4.0-SNAPSHOT</version>
+        </dependency>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>vaadin-button-testbench</artifactId>
+            <version>4.0-SNAPSHOT</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Would it be possible in the ITs to use `com.vaadin.flow.component.html.NativeButton` instead of `com.vaadin.flow.component.button.Button` to reduce the inter-componetn dependencies (and also other ITs seem to use `NativeButton`).

Reply:
Yes, will do

Reply:
done